docker/controller/start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Only write the Turbinia configuration file if TURBINIA_CONF is not empty and the file does not exist (or is 0 bytes in size)
+if [ ! -z "$TURBINIA_CONF" ] && [ ! -s /etc/turbinia/turbinia.conf ]
+then
+    echo "${TURBINIA_CONF}" | base64 -d > /etc/turbinia/turbinia.conf
+fi
+
+if [ ! -z ${TURBINIA_OUTPUT_DIR} ] && [ ! -d ${TURBINIA_OUTPUT_DIR} ]
```

Comment:
Will the output or tmp dirs be used on this host?

Reply:
No actually will remove since not needed